### PR TITLE
Use the 2.13 ACM in the governance-policy-framework main branch

### PR DESCRIPTION
**Pull request by mprahl**

### CODE REVIEW

File: ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ tests:
     dependencies:
       COMPONENT_IMAGE_REF: grc-policy-framework-tests
     env:
-      ACM_RELEASE_VERSION: release-2.12
+      ACM_RELEASE_VERSION: release-2.13
```
Comment (dhaiducek) [inline ci-operator/config/stolostron/governance-policy-framework/stolostron-governance-policy-framework-main.yaml:36]: Set `PIPELINE_STAGE: integration` also. Without BVTs running, just setting the version isn't going to do it.

**Review by dhaiducek — CHANGES_REQUESTED** [review]

**Review by dhaiducek — APPROVED** [review]
Here goes nothing! 😅

**Comment by mprahl** [comment]
@dhaiducek the test failure in the rehearsal was an OperatorPolicy flake and got past the issue I was encountering before so I'll override the failure.

**Pull request merged**